src/components/comments/dto/list-comment.dto.ts

```diff
@@ -0,0 +1,38 @@
+import { Field, InputType, ObjectType } from '@nestjs/graphql';
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import {
+  ID,
+  Order,
+  PaginatedList,
+  SecuredList,
+  SortablePaginationInput,
+} from '../../../common';
+import { Comment } from './comment.dto';
+
+@InputType()
+export abstract class CommentFilters {
+  @Field()
+  readonly threadId: ID;
```

Comment:
I think this filter should be internal, since we'll connect it manually in a resolver.